content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+
+Second, I try to adjust my perspective as a teacher, understanding that
+
+- *I am operating on stolen land.*
+- *My students likely have an inseparable connection to their communities and their land.* 
+- *My students are strong and resilient in the face of intergenerational trauma.*^[[The National Native American Boarding School Healing Coalition - Impact of Historical Trauma](https://boardingschoolhealing.org/education/impact-of-historical-trauma/)]
+- *My own notions of health, medicine, and justice are highly individualistic and institutional.*
+
+Then concretely, I put the following principles into action.
+
+- *The destination is clear*: Roadmapping is central to effective teaching in any classroom. One of the biggest barriers for students are the tacit expectations teachers place on them, including why they are taking the class and their performance and behavior in it. Students should have a clear outline about what we're doing throughout and at each stage of the course: what they will learn from it, how and when their performance will be evaluated, and how it will be useful to them outside the classroom. Yup, a good written syllabus + explicit and regular expression of expectations and learning objectives are foundational for inclusive classroms. Doing this well takes time, practice, and planning. ^[[Learning Objectives and Backward Design](https://tep.uoregon.edu/learning-objectives-and-backward-design)]
+
+- *Everyone is up to speed*: At the end of any 15-20 minute lecture chunk I use low-stakes, formative assessments (e.g. ungraded quizes^[[Formative vs. Summative Assessments: What's the Difference?](https://www.aeseducation.com/blog/formative-vs.-summative-assessments-what-do-they-mean)]) to focus everyone's attention on the big picture of what we've just talked about and track/realign any misconceptions.
+
+- *Representation matters*: For slideshows, I aim to highlight Native people that are working to solve the problems showcased in each dataset. Whenever possible, I also choose pictures of native people applying the skills that the students are learning. Each of these gives students role models, so they can envision themselves doing data analysis for public health and social justice.
+
+- *Make the connection*: Upon choosing their data set and at mileposts during their analysis, I ask students to express how the data relates to them and their communities, and how they can share insights from the data with others. Letting students tell their stories builds trust. In the future, I plan to use a talking stick as each speaker shares.
+
+- *You have agency*: I let students decide on their project datasets.
```

Comment:
I'd like to know more about how students find datasets and whether the author steers them toward problems and data that are doable.